### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by expede**
[:eyes: Preview](https://github.com/RhizomeDB/pomo-ipld/blob/v0.1/README.md)

### CODE REVIEW

**Comment by expede** [comment]
I think that's good enough. It's pretty simple for this much text 😛

**Review by QuinnWilton — APPROVED** [review]
This looks good to me! Is there anything in particular here that you'd like specific feedback on?

File: README.md [hunk 1]
```diff
@@ -13,22 +13,167 @@
...
+[DAG-CBOR]: https://ipld.io/specs/codecs/dag-cbor/spec/
+[DAG-JSON]: https://ipld.io/docs/codecs/known/dag-json/
+[Entity ID]: #212-entity-id
+[Fact]: #211-fact
 [Fission Codes]: https://fission.codes
+[IEEE 754-2019]: https://en.wikipedia.org/wiki/IEEE_754
+[IPFS]: https://ipfs.io
+[IPLD]: https://ipld.io
+[Multicodec]: https://github.com/multiformats/multicodec
+[Multihash]: https://multiformats.io/multihash/
```
Comment (QuinnWilton) [inline README.md]: I just realized this link is broken now 😅 It's the same one they link on their homepage, so it's on their end, but I don't know if there's a better place for us to reference? Maybe the github repo, like we do for multicodec?

https://github.com/multiformats/multihash

Comment (expede) [reply README.md]: Wow that's not great! That definely loaded yesterday 😬 

Will update; thanks!

**Comment by expede** [comment]
> This looks good to me! Is there anything in particular here that you'd like specific feedback on?

@QuinnWilton I think it's a pretty simple spec, so I'm not surprised that there aren't big contentous issues. I guess the main thing is some people really care about maps vs tuples for extensability, but I don't think that's going to be a problem for us (she says, tempting fate)

**Comment by QuinnWilton** [comment]
Makes sense! Well, I've always been one to tempt fate, so what you have here looks good to me :)

**Pull request merged**